### ISSUE TITLE
Gem

### ISSUE THREAD

**Issue by LandonSchropp** [jamiew/tumblr-photo-downloader#10]
It would be more convenient for me for a project I'm working on if this were packaged up as a gem. Would you accept a pull request that turned this repo into a gem?

**Comment by kristopolous** [comment]
oh man it's not really that kind of thing ... what is the idea?

**Comment by kristopolous** [comment]
i mean let me be clearer ... it's a bunch of mostly self-contained ruby scripts without a publicly exposed api.  I don't think the video part is working any more ... this was mostly for a deep learning engine I was writing.

**Comment by kristopolous** [comment]
sorry about those last two ... I didn't notice this was jamiew's version and not mine.  I deleted them.

**Comment by kristopolous** [comment]
I think that the thing of relevant I *should* speak of is that my variation is far faster then jamiew's version ... I put weeks into development of it ... https://github.com/kristopolous/tumblr-scraper it's about 30-50 times faster

**Comment by jamiew** [comment]
@LandonSchropp yes that seems cool. Is that so you can include the code or to use it as a binary?

@kristopolous your scraper is cool too, I would be happy to point to it from my README. This repository works but isn't particularly fancy

**Comment by jamiew** [comment]
@LandonSchropp any interest in gem'ifying this? Woudl Would love to see that

**Comment by LandonSchropp** [comment]
@jamiew Unfortunately I'm swamped with other work right now. I might be able to get to turning this into a gem, but I can't say when.

**Comment by jamiew** [comment]
Sure no problem at all. I'm gonna leave this open since I think it's a
great idea
